### PR TITLE
let end users fix themselves when they need to re-install the SlashDeploy app.

### PR DESCRIPTION
**Pull request by russellballestrini**
I think this will help end users fix themselves when they need to re-install the SlashDeploy `Slack` application.

	modified:   app/models/slack_bot.rb

### CODE REVIEW

File: app/models/slack_bot.rb [hunk 1]
```diff
@@ -3,9 +3,13 @@ class SlackBot < ActiveRecord::Base
 
   def self.from_auth_hash(auth_hash)
     bot_info = auth_hash[:extra][:bot_info]
-    find_or_create_by(id: bot_info[:bot_user_id]) do |bot|
+    bot = find_or_create_by(id: bot_info[:bot_user_id]) do |bot|
```
Comment (ejholmes) [inline app/models/slack_bot.rb]: This all might read a little better if it was just turned into:

```ruby
bot = find_by(id: bot_info[:bot_user_id])
if bot
  bot.update_attributes(...)
else
  create(...)
end
```

Comment (russellballestrini) [reply app/models/slack_bot.rb]: fixed I think.

**Comment by russellballestrini** [comment]
@ejholmes - could you help me locate the proper place to add a test for this flow?

**Comment by russellballestrini** [comment]
After some messing with slashdeploy, I found that the proper way for an end user to reconnect with github is to go here:

- https://slashdeploy.io/auth/github

**Pull request merged**